grub2-efi/install

```diff
@@ -11,17 +11,23 @@ case "$target" in
   i?86 ) target=i386 ;;
   x86_64 | amd64 ) target=x86_64 ;;
   aarch64 ) target=arm64 ;;
-  armv7l ) target=arm ;;
+  arm* ) target=arm ;;
 esac
 
 target="$target-efi"
 
 echo "target = $target"
 
+# Systems without efivars should just fall back to install grub as removable
```

Comment:
> we do not have NVRAM because we would have to store that on the same flash that Linux may be running on, creating device ownership conflicts

Does not U-Boot itself have some configuration space? Can it be used to store "EFI" variables?